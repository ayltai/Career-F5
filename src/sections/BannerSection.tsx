import { Place as PlaceIcon, Schedule as ScheduleIcon, WorkOutline as WorkIcon, } from '@mui/icons-material';
import { Box, Container, Stack, Typography, useMediaQuery, } from '@mui/material';
import React, { ReactNode, } from 'react';
import { useTranslation, } from 'react-i18next';

import type { Section, } from '../@types';

export const BannerSection = ({
    title,
    description,
    background,
    foregroundImage,
    callToAction,
} : Section & {
    background?      : string,
    foregroundImage? : string,
    callToAction?    : ReactNode,
}) => {
    const desktopMode = useMediaQuery('(min-width:900px)');

    const { t, } = useTranslation();

    const info = (
        <Stack
            alignItems={desktopMode ? undefined : 'center'}
            direction='column'>
            <Typography
                display='flex'
                alignItems='center'
                color='text.secondary'
                variant='body2'>
                <ScheduleIcon />&nbsp;{t('banner_part_time')}
            </Typography>
            <Typography
                display='flex'
                alignItems='center'
                color='text.secondary'
                variant='body2'>
                <PlaceIcon />&nbsp;{t('banner_remote')}
            </Typography>
            <Typography
                display='flex'
                alignItems='center'
                color='text.secondary'
                variant='body2'>
                <WorkIcon />&nbsp;{t('banner_interview')}
            </Typography>
        </Stack>
    );

    return desktopMode ? (
        <Box
            sx={{
                background,
            }}
            height={512}>
            <Container maxWidth='lg'>
                <Stack
                    paddingLeft={2}
                    direction='row'>
                    <Stack
                        width='60%'
                        direction='column'
                        justifyContent='center'>
                        <Typography
                            gutterBottom
                            variant='h2'
                            fontWeight='bold'>
                            {title}
                        </Typography>
                        <Typography
                            gutterBottom
                            variant='h6'>
                            {description}
                        </Typography>
                        {callToAction && (
                            <Box paddingY={3}>
                                {callToAction}
                            </Box>
                        )}
                        {info}
                    </Stack>
                    <Box
                        width='40%'
                        display='flex'>
                        <Box
                            height={512}
                            marginX='auto'
                            marginRight={0}
                            alignItems='center'>
                            <img
                                style={{
                                    marginTop : 32,
                                }}
                                width={317}
                                height={480}
                                alt={title}
                                src={foregroundImage} />
                        </Box>
                    </Box>
                </Stack>
            </Container>
        </Box>
    ) : (
        <Box sx={{
            background,
        }}>
            <Stack
                padding={2}
                direction='column'
                justifyContent='center'
                textAlign='center'>
                <Typography
                    gutterBottom
                    variant='h2'
                    fontWeight='bold'>
                    {title}
                </Typography>
                <Typography
                    gutterBottom
                    variant='h6'>
                    {description}
                </Typography>
                <Box
                    paddingTop={2}
                    paddingBottom={3}>
                    {callToAction}
                </Box>
                {info}
                <Box
                    height={512}
                    marginX='auto'
                    marginBottom={-2}
                    alignItems='center'>
                    <img
                        style={{
                            marginTop : 32,
                        }}
                        width={317}
                        height={480}
                        alt={title}
                        src={foregroundImage} />
                </Box>
            </Stack>
        </Box>
    );
};
