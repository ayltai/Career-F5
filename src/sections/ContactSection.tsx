import { Box, Container, Stack, Typography, useMediaQuery, } from '@mui/material';
import React, { ReactNode, } from 'react';

import type { Section, } from '../@types';
import { FormGenerator, FormGeneratorProps, } from '../components/FormGenerator';

export const ContactSection = ({
    background,
    category,
    title,
    description,
    definition,
    contactAction,
    onSubmit,
} : Section & FormGeneratorProps & {
    background?    : string,
    category?      : string,
    contactAction? : ReactNode,
}) => {
    const desktopMode = useMediaQuery('(min-width:900px)');

    return (
        <Box
            sx={{
                background,
            }}
            paddingY={8}>
            <Container maxWidth='lg'>
                <Stack
                    padding={desktopMode ? 2 : 1}
                    direction={desktopMode ? 'row' : 'column'}>
                    <Stack
                        width={desktopMode ? '50%' : '100%'}
                        padding={desktopMode ? 2 : 0}
                        direction='column'>
                        <Typography
                            gutterBottom
                            color='secondary'
                            variant='subtitle1'
                            fontWeight='bold'>
                            {category}
                        </Typography>
                        <Typography
                            gutterBottom
                            variant='h3'
                            fontWeight='bold'>
                            {title}
                        </Typography>
                        <Typography variant='h6'>
                            {description}
                        </Typography>
                        {contactAction && (
                            <Box paddingY={4}>
                                {contactAction}
                            </Box>
                        )}
                    </Stack>
                    <Box
                        width={desktopMode ? '50%' : '100%'}
                        padding={desktopMode ? 2 : 0}>
                        <FormGenerator
                            definition={definition}
                            onSubmit={onSubmit} />
                    </Box>
                </Stack>
            </Container>
        </Box>
    );
};
