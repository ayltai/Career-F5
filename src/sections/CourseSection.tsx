import { Box, Card, CardActions, CardContent, Chip, Container, Grid, Stack, Typography, } from '@mui/material';
import React from 'react';
import { useTranslation, } from 'react-i18next';

import type { Course, Section, } from '../@types';

export const CourseSection = ({
    backgroundColor,
    category,
    title,
    courses,
} : Section & {
    backgroundColor? : string,
    courses          : Course[],
}) => {
    const { t, } = useTranslation();

    return (
        <Box
            sx={{
                backgroundColor,
            }}
            paddingTop={8}
            paddingBottom={4}>
            <Container maxWidth='lg'>
                <Stack
                    direction='column'
                    textAlign='center'>
                    <Typography
                        gutterBottom
                        color='secondary'
                        variant='subtitle1'
                        fontWeight='bold'>
                        {category}
                    </Typography>
                    <Container maxWidth='md'>
                        <Typography
                            variant='h3'
                            fontWeight='bold'>
                            {title}
                        </Typography>
                    </Container>
                    <Box marginY={8}>
                        <Grid
                            container
                            rowSpacing={{
                                xs : 3,
                                sm : 4,
                                md : 5,
                            }}
                            columnSpacing={{
                                xs : 1,
                                sm : 2,
                                md : 3,
                            }}
                            justifyContent='space-around'>
                            {courses.map(course => (
                                <Grid
                                    key={course.route || course.text}
                                    sx={{
                                        display : 'flex',
                                    }}
                                    item
                                    xs={12}
                                    md={3}>
                                    <Card
                                        sx={{
                                            display       : 'flex',
                                            flexDirection : 'column',
                                            ':hover'      : {
                                                boxShadow : 8,
                                            },
                                        }}
                                        elevation={2}>
                                        <CardContent sx={{
                                            padding       : 2,
                                            display       : 'flex',
                                            flexDirection : 'column',
                                            flexGrow      : 1,
                                        }}>
                                            <Typography
                                                variant='subtitle1'
                                                color='text.secondary'>
                                                {t(course.category)}
                                            </Typography>
                                            <Typography variant='h5'>
                                                {course.text && t(course.text)}
                                            </Typography>
                                        </CardContent>
                                        {course.tags && (
                                            <CardActions sx={{
                                                padding : 2,
                                            }}>
                                                <Stack
                                                    useFlexGap
                                                    spacing={1}
                                                    direction='row'
                                                    flexWrap='wrap'
                                                    justifyContent='flex-start'>
                                                    {course.tags.map(tag => (
                                                        <Chip
                                                            key={tag}
                                                            size='small'
                                                            label={tag} />
                                                    ))}
                                                </Stack>
                                            </CardActions>
                                        )}
                                    </Card>
                                </Grid>
                            ))}
                        </Grid>
                    </Box>
                </Stack>
            </Container>
        </Box>
    );
};
