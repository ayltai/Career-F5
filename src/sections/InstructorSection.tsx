import { OpenInNew as OpenInNewIcon, } from '@mui/icons-material';
import { Avatar, Box, Container, Grid, Link, Stack, Typography, useMediaQuery, } from '@mui/material';
import mixpanel from 'mixpanel-browser';
import React, { Fragment, } from 'react';
import { useTranslation, } from 'react-i18next';

import type { Section, } from '../@types';
import { experiences, } from '../data';

export const InstructorSection = ({
    category,
    title,
} : Section) => {
    const desktopMode = useMediaQuery('(min-width:900px)');

    const { t, } = useTranslation();

    const descriptions : string[] = t('instructor_descriptions', {
        returnObjects : true,
    });

    const handleClick = () => {
        if (process.env.REACT_APP_MIXPANEL_TOKEN) mixpanel.track('Referral/LinkedIn');

        return true;
    };

    const heading = (
        <Fragment>
            <Typography
                gutterBottom
                marginTop={4}
                color='secondary'
                variant='subtitle1'
                fontWeight='bold'>
                {category}
            </Typography>
            <Typography
                variant='h3'
                fontWeight='bold'>
                {title}
            </Typography>
        </Fragment>
    );

    const avatar = (
        <Avatar
            sx={{
                width      : desktopMode ? 480 : '100%',
                height     : desktopMode ? 480 : '100%',
                margin     : desktopMode ? 8 : undefined,
                marginLeft : desktopMode ? 4 : undefined,
                marginTop  : desktopMode ? undefined : 4,
            }}
            alt={t<string>('instructor_header')}
            src='/images/Profile-Business-Small.webp' />
    );

    const logos = (
        <Grid
            container
            marginBottom={4}
            rowSpacing={{
                xs : 3,
                sm : 4,
                md : 5,
            }}
            columnSpacing={{
                xs : 1,
                sm : 2,
                md : 3,
            }}
            justifyContent='space-around'>
            {experiences.map(experience => (
                <Grid
                    key={experience.key}
                    container
                    item
                    xs={6}
                    alignItems='center'
                    textAlign='center'>
                    <Box sx={{
                        marginX : 'auto',
                    }}>
                        <img
                            style={{
                                width     : desktopMode ? 180 : 120,
                                objectFit : desktopMode ? 'contain' : 'scale-down',
                            }}
                            alt={t<string>(experience.key)}
                            src={experience.path} />
                    </Box>
                </Grid>
            ))}
        </Grid>
    );

    const description = (
        <Typography
            marginTop={4}
            variant='body1'>
            <Link
                underline='hover'
                href='https://www.linkedin.com/in/ayltai/'
                target='_blank'
                onClick={handleClick}>
                {t('instructor_name')}
                <OpenInNewIcon fontSize='small' />
            </Link>
            {descriptions.map(desc => (
                <Fragment key={desc}>
                    {desc}<br /><br />
                </Fragment>
            ))}
        </Typography>
    );

    return (
        <Container maxWidth='lg'>
            {desktopMode && (
                <Grid container>
                    <Grid
                        item
                        xs={6}>
                        <Box
                            width='100%'
                            flexDirection='column'>
                            {avatar}
                            {logos}
                        </Box>
                    </Grid>
                    <Grid
                        item
                        paddingX={8}
                        paddingTop={2}
                        paddingBottom={8}
                        xs={6}
                        display='flex'
                        flexDirection='column'
                        justifyContent='center'>
                        {heading}
                        {description}
                    </Grid>
                </Grid>
            )}
            {!desktopMode && (
                <Stack
                    direction='column'
                    justifyContent='center'
                    textAlign='center'>
                    {heading}
                    {avatar}
                    {description}
                    {logos}
                </Stack>
            )}
        </Container>
    );
};
