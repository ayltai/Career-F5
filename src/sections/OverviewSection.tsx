import { Box, Container, Stack, Typography, useMediaQuery ,} from '@mui/material';
import React from 'react';

import type { Section, } from '../@types';

export const OverviewSection = ({
    category,
    title,
    description,
    image,
} : Section & {
    image : string,
}) => {
    const desktopMode = useMediaQuery('(min-width:900px)');

    return desktopMode ? (
        <Box
            width='100%'
            marginY={8}
            height={512}>
            <Container maxWidth='lg'>
                <Stack direction='row'>
                    <Box
                        width='40%'
                        display='flex'>
                        <Box
                            height={512}
                            marginX='auto'
                            alignItems='center'>
                            <img
                                style={{
                                    borderRadius : 32,
                                }}
                                width={314}
                                height={480}
                                alt={'dummy'}
                                src={image} />
                        </Box>
                    </Box>
                    <Stack
                        width='60%'
                        marginTop={-4}
                        paddingRight={8}
                        direction='column'
                        justifyContent='center'>
                        <Typography
                            gutterBottom
                            color='secondary'
                            variant='subtitle1'
                            fontWeight='bold'>
                            {category}
                        </Typography>
                        <Typography
                            gutterBottom
                            variant='h3'
                            fontWeight='bold'>
                            {title}
                        </Typography>
                        <Typography variant='body1'>
                            {description}
                        </Typography>
                    </Stack>
                </Stack>
            </Container>
        </Box>
    ) : (
        <Box width={'100%'}>
            <Stack
                padding={2}
                direction='column'
                justifyContent='center'
                textAlign='center'>
                <Typography
                    gutterBottom
                    color='secondary'
                    variant='subtitle1'
                    fontWeight='bold'>
                    {category}
                </Typography>
                <Typography
                    gutterBottom
                    variant='h3'
                    fontWeight='bold'>
                    {title}
                </Typography>
                <Typography variant='body1'>
                    {description}
                </Typography>
                <Box
                    height={512}
                    marginX='auto'
                    marginY={4}
                    alignItems='center'>
                    <img
                        style={{
                            borderRadius : 32,
                        }}
                        width={314}
                        height={480}
                        alt={'dummy'}
                        src={image} />
                </Box>
            </Stack>
        </Box>
    );
};
