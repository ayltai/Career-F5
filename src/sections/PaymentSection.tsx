import { Box, Container, Stack, Typography, } from '@mui/material';
import React from 'react';
import { useTranslation, } from 'react-i18next';

import type { Section, } from '../@types';
import { PaymentPlan, } from '../components';

export const PaymentSection = ({
    category,
    title,
    description,
} : Section) => {
    const { t, } = useTranslation();

    const plans = [
        {
            title       : t<string>('payment_full_title'),
            description : t<string>('payment_full_description'),
            cost        : t<string>('payment_full_cost'),
        },
        {
            title       : t<string>('payment_instalment_title'),
            description : t<string>('payment_instalment_description'),
            cost        : t<string>('payment_instalment_cost'),
        },
        {
            title       : t<string>('payment_discount_title'),
            description : t<string>('payment_discount_description'),
            cost        : t<string>('payment_discount_cost'),
        },
        {
            title       : t<string>('payment_interview_title'),
            description : t<string>('payment_interview_description'),
            cost        : t<string>('payment_interview_cost'),
        },
    ];

    return (
        <Box paddingY={8}>
            <Container maxWidth='lg'>
                <Stack
                    spacing={2}
                    direction='column'>
                    <Typography
                        gutterBottom
                        color='secondary'
                        variant='subtitle1'
                        fontWeight='bold'>
                        {category}
                    </Typography>
                    <Box marginBottom={2}>
                        <Typography
                            gutterBottom
                            variant='h3'
                            fontWeight='bold'>
                            {title}
                        </Typography>
                    </Box>
                    <Typography
                        gutterBottom
                        variant='body1'>
                        {description}
                    </Typography>
                    {plans.map(plan => (
                        <Box
                            key={plan.title}
                            paddingTop={4}>
                            <PaymentPlan {...plan} />
                        </Box>
                    ))}
                </Stack>
            </Container>
        </Box>
    );
};
