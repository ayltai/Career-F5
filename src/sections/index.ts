export { BannerSection, } from './BannerSection';
export { ContactSection, } from './ContactSection';
export { CourseSection, } from './CourseSection';
export { InstructorSection, } from './InstructorSection';
export { OverviewSection, } from './OverviewSection';
export { PaymentSection, } from './PaymentSection';
export { ProspectSection, } from './ProspectSection';
export { StructureSection, } from './StructureSection';
