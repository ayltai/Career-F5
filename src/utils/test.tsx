import { ThemeProvider, } from '@mui/material';
import { configureStore, PreloadedState, Store, } from '@reduxjs/toolkit';
import '@testing-library/jest-dom/extend-expect';
import { render, } from '@testing-library/react';
import mediaQuery from 'css-mediaquery';
import React, { ReactNode, } from 'react';
import { Provider, } from 'react-redux';

import { preferencesReducer, } from '../states';
import { AppTheme, } from '../styles';

export const createMatchMedia = (width : number) => (query : string) => ({
    matches             : mediaQuery.match(query, {
        width,
    }),
    media               : '',
    onchange            : null,
    addListener         : () => {},
    removeListener      : () => {},
    addEventListener    : () => {},
    removeEventListener : () => {},
    dispatchEvent       : () => false,
});

const createStore = (preloadedState? : Record<string, any>) => configureStore({
    reducer : {
        preferences : preferencesReducer,
    },
    preloadedState,
});

const defaultStore = createStore();

type RootState = ReturnType<typeof defaultStore.getState>;

const customRender = (ui : any, {
    preloadedState,
    store = createStore(preloadedState),
    ...rest
} : {
    preloadedState?  : PreloadedState<RootState>,
    store?           : Store<RootState>,
    [ rest : string] : any,
} = {}) => render(ui, {
    wrapper : ({
        children,
    } : {
        children : ReactNode,
    }) => (
        <Provider store={store}>
            <ThemeProvider theme={AppTheme('en')}>
                {children}
            </ThemeProvider>
        </Provider>
    ),
    ...rest,
});

export * from '@testing-library/react';

export { customRender as render, };
