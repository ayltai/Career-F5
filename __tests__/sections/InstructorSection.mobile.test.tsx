import React from 'react';

import { createMatchMedia, render, } from '../../src/utils/test';
import { InstructorSection, } from '../../src/sections/InstructorSection';

describe('InstructorSection', () => {
    beforeAll(() => {
        window.matchMedia = createMatchMedia(360);
    });

    it('renders correctly for desktop', () => expect(render(
        <InstructorSection
            category='category'
            title='title' />
    )).toMatchSnapshot());
});
