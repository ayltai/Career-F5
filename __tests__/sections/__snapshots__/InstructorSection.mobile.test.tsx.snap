// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InstructorSection renders correctly for desktop 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
      >
        <div
          class="MuiStack-root css-153idv-MuiStack-root"
        >
          <h6
            class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-gutterBottom css-16ra52y-MuiTypography-root"
          >
            category
          </h6>
          <h3
            class="MuiTypography-root MuiTypography-h3 css-1uokl1g-MuiTypography-root"
          >
            title
          </h3>
          <div
            class="MuiAvatar-root MuiAvatar-circular css-b6t4zq-MuiAvatar-root"
          >
            <img
              alt="instructor_header"
              class="MuiAvatar-img css-1pqm26d-MuiAvatar-img"
              src="/images/Profile-Business-Small.webp"
            />
          </div>
          <p
            class="MuiTypography-root MuiTypography-body1 css-159jqdy-MuiTypography-root"
          >
            <a
              class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1jcp6fe-MuiTypography-root-MuiLink-root"
              href="https://www.linkedin.com/in/ayltai/"
              target="_blank"
            >
              instructor_name
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
                data-testid="OpenInNewIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
                />
              </svg>
            </a>
            line 1
            <br />
            <br />
            line 2
            <br />
            <br />
          </p>
          <div
            class="MuiGrid-root MuiGrid-container css-1wb8vpz-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-wbqxy0-MuiGrid-root"
            >
              <div
                class="MuiBox-root css-a9n7s9"
              >
                <img
                  alt="experience_jpmc"
                  src="/images/jpmc.webp"
                  style="width: 120px; object-fit: scale-down;"
                />
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-wbqxy0-MuiGrid-root"
            >
              <div
                class="MuiBox-root css-a9n7s9"
              >
                <img
                  alt="experience_hsbc"
                  src="/images/hsbc.webp"
                  style="width: 120px; object-fit: scale-down;"
                />
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-wbqxy0-MuiGrid-root"
            >
              <div
                class="MuiBox-root css-a9n7s9"
              >
                <img
                  alt="experience_mox"
                  src="/images/mox.webp"
                  style="width: 120px; object-fit: scale-down;"
                />
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-wbqxy0-MuiGrid-root"
            >
              <div
                class="MuiBox-root css-a9n7s9"
              >
                <img
                  alt="experience_chc"
                  src="/images/chc.webp"
                  style="width: 120px; object-fit: scale-down;"
                />
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-wbqxy0-MuiGrid-root"
            >
              <div
                class="MuiBox-root css-a9n7s9"
              >
                <img
                  alt="experience_nhs"
                  src="/images/nhs.webp"
                  style="width: 120px; object-fit: scale-down;"
                />
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-wbqxy0-MuiGrid-root"
            >
              <div
                class="MuiBox-root css-a9n7s9"
              >
                <img
                  alt="experience_generali"
                  src="/images/generali.webp"
                  style="width: 120px; object-fit: scale-down;"
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="MuiStack-root css-153idv-MuiStack-root"
      >
        <h6
          class="MuiTypography-root MuiTypography-subtitle1 MuiTypography-gutterBottom css-16ra52y-MuiTypography-root"
        >
          category
        </h6>
        <h3
          class="MuiTypography-root MuiTypography-h3 css-1uokl1g-MuiTypography-root"
        >
          title
        </h3>
        <div
          class="MuiAvatar-root MuiAvatar-circular css-b6t4zq-MuiAvatar-root"
        >
          <img
            alt="instructor_header"
            class="MuiAvatar-img css-1pqm26d-MuiAvatar-img"
            src="/images/Profile-Business-Small.webp"
          />
        </div>
        <p
          class="MuiTypography-root MuiTypography-body1 css-159jqdy-MuiTypography-root"
        >
          <a
            class="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineHover css-1jcp6fe-MuiTypography-root-MuiLink-root"
            href="https://www.linkedin.com/in/ayltai/"
            target="_blank"
          >
            instructor_name
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall css-ptiqhd-MuiSvgIcon-root"
              data-testid="OpenInNewIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 19H5V5h7V3H5c-1.11 0-2 .9-2 2v14c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2v-7h-2v7zM14 3v2h3.59l-9.83 9.83 1.41 1.41L19 6.41V10h2V3h-7z"
              />
            </svg>
          </a>
          line 1
          <br />
          <br />
          line 2
          <br />
          <br />
        </p>
        <div
          class="MuiGrid-root MuiGrid-container css-1wb8vpz-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-wbqxy0-MuiGrid-root"
          >
            <div
              class="MuiBox-root css-a9n7s9"
            >
              <img
                alt="experience_jpmc"
                src="/images/jpmc.webp"
                style="width: 120px; object-fit: scale-down;"
              />
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-wbqxy0-MuiGrid-root"
          >
            <div
              class="MuiBox-root css-a9n7s9"
            >
              <img
                alt="experience_hsbc"
                src="/images/hsbc.webp"
                style="width: 120px; object-fit: scale-down;"
              />
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-wbqxy0-MuiGrid-root"
          >
            <div
              class="MuiBox-root css-a9n7s9"
            >
              <img
                alt="experience_mox"
                src="/images/mox.webp"
                style="width: 120px; object-fit: scale-down;"
              />
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-wbqxy0-MuiGrid-root"
          >
            <div
              class="MuiBox-root css-a9n7s9"
            >
              <img
                alt="experience_chc"
                src="/images/chc.webp"
                style="width: 120px; object-fit: scale-down;"
              />
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-wbqxy0-MuiGrid-root"
          >
            <div
              class="MuiBox-root css-a9n7s9"
            >
              <img
                alt="experience_nhs"
                src="/images/nhs.webp"
                style="width: 120px; object-fit: scale-down;"
              />
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-item MuiGrid-grid-xs-6 css-wbqxy0-MuiGrid-root"
          >
            <div
              class="MuiBox-root css-a9n7s9"
            >
              <img
                alt="experience_generali"
                src="/images/generali.webp"
                style="width: 120px; object-fit: scale-down;"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
